tst/regression/test_suites/particle_leapfrog_outflow/particle_leapfrog_outflow.py

```diff
@@ -36,19 +36,22 @@ def Analyse(self, parameters):
         data = np.genfromtxt("particles.csv", delimiter=",", names=True)
 
         # pick last cycle (given current parameter file)
-        final_data = data[data["ncycle"] == 24]
+        final_data = data[data["ncycle"] == 38]
         final_data.sort(order="particles_id")
 
         # see examples/particle_leapfrog/particle_leapfrog.cpp for reference data
         # here, outflow boundaries are used such that some of the particles escape
         # this tests code paths not otherwise present (when blocks have >0 neighbors)
         ref_data = np.array(
             [
-                [-0.1, 0.3, 0.3353164693, 0.0, 0.0, 0.5],
-                [0.3906329387, 0.4706329387, -0.0293670613, 1.0, 1.0, 1.0],
+                [0.4, 0.2, 0.3, 1.0, 0.0, 0.0],
+                [0.4, 0.4, 0.3, 0.0, 1.0, 0.0],
+                [-0.1, 0.3, 0.45, 0.0, 0.0, 0.5],
             ]
```

Comment:
I see why you changed the answer here in combination with the `tlim=0.5` in the input file.
Looks like the original test was not working correctly.
Do you think it's worth updating the test case (and, thus, also `particle_leapfrog.cpp`) to include more particles that pass through all boundaries? Just to make sure everything works as expected?

Reply:
Yes this is a good idea. I added particles that test all corner and face crossings (not edge crossings). Golds updated, results look good to me.

Reply:
Looks also good to me! I'll trigger the full suite now to to make sure.